k8s/crd-samples/deployment-cstor-pool.yaml

```diff
@@ -53,5 +65,5 @@ spec:
           type: Directory
       - name: tmp
         hostPath:
-          path: /tmp
+          path: /var/openebs/shared-a2b
```

Comment:
Is this name a magic name?
If yes, the reason needs to be provided.
If no, please put a name that is understood easily.

Reply:
Since this is WIP and we have decided to move them to openebs/elves, I'll address the review comments in a separate PR. @kmova  @AmitKumarDas